r_test_requirements.R

```diff
@@ -1,8 +1,12 @@
-# note: fixest and broom are installed via conda
+# note: R, fixest, sandwich, broom are installed via conda
 install.packages(
-    c('fixest', 'broom','clubSandwich', 'did2s', 'wildrwolf', 'reticulate', 'ivDiag', 'stats', 'base', 'car'),
+    c('did2s', 'reticulate', 'ivDiag', 'car'),
```

Comment:
We should also be able to install [r-car via conda-forge](https://anaconda.org/r/r-car).